project_event/i18n/fr.po

```diff
@@ -656,7 +663,7 @@ msgstr "Wizard de création d'activité d'événement à partir d'un modèle"
 #. module: project_event
 #: model:ir.model,name:project_event.model_project_event_wizard
 msgid "Wizard Event Creation from Template"
-msgstr "Wizard de création d'événement à partir d'un modèle"
+msgstr ""
```

Comment:
@jananjoy why you delete the old translation : Wizard de création d'événement à partir d'un modèle

Reply:
@rimbendhaou : Update  the  translation to the fr.po file